sign.sh

```diff
@@ -205,4 +233,13 @@ signedArchive="${TMP_DIR}/OpenJDK${archiveExtension}"
 
 cd "${WORKSPACE}"
 mv "${signedArchive}" "${ARCHIVE}"
+
+if ([ "$OPERATING_SYSTEM" = "aix" ] || [ "$OPERATING_SYSTEM" = "linux" ]) && [ "$SIGN_TOOL" = "ucl" ]; then
+  # sign the tarball
+  echo "Sign archive ${ARCHIVE}"
+  ucl sign --hash SHA256 -n WindowsSHA -i "${ARCHIVE}" -o "${ARCHIVE}.sig"
```

Comment:
I'd like to see `WindowsSha` replaced with `SIGNING_CERTIFICATE` (which is usually a filepath). That will make this change easier to submit to Adopt. I'm fine with it for now since we already do it for Windows.

Reply:
It will be addressed in a different PR.